control_panel_api/settings/base.py

```diff
@@ -144,10 +145,15 @@ def is_enabled(value):
 K8S_WORKER_ROLE_NAME = os.environ.get('K8S_WORKER_ROLE_NAME', '')
 SAML_PROVIDER = os.environ.get('SAML_PROVIDER', '')
 
-RAVEN_CONFIG = {
-    'dsn': os.environ.get('SENTRY_DSN', ''),
-    'environment': ENV,
-}
+if os.environ.get('SENTRY_DSN'):
+    this_is_running_in_the_django_shell = "shell" in sys.argv
+    RAVEN_CONFIG = {
+        'dsn': os.environ.get('SENTRY_DSN', ''),
+        'environment': ENV,
+        'ignore_exceptions': ('*') if this_is_running_in_the_django_shell else (),
```

Comment:
`('*')` is not a tuple. Is it really what you want?

Reply:
Thanks @est! You're right. I'll make it a list `['*']`